website/docs/community/bindings.md

```diff
@@ -17,7 +17,9 @@ As a C library, bindings can be made to call H3 functions from different program
 
 ## BigQuery
 
-- [cartodb/bigquery-jslibs](https://github.com/CartoDB/bigquery-jslibs)
+- [CARTO Analytics Toolbox](https://docs.carto.com/analytics-toolbox-bigquery/sql-reference/h3/)
+- [CARTO Analytics Toobox source code](https://github.com/CartoDB/analytics-toolbox-core)
+- [cartodb/bigquery-jslibs (Deprecated)](https://github.com/CartoDB/bigquery-jslibs) 
```

Comment:
Is there a reason to include this if it's deprecated?

Reply:
No, no reason, good point.